.gitignore

```diff
@@ -1,7 +1,6 @@
 node_modules/
 .sketchpad/
-/index.js
+/index.*
```

Comment:
This will also catch `index.d.ts` file 😄

Reply:
Ah I guess we don't need to commit it

Reply:
yep, that's what I thought.